x/validator-vesting/internal/keeper/querier.go

```diff
@@ -0,0 +1,62 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+	authexported "github.com/cosmos/cosmos-sdk/x/auth/exported"
+	"github.com/cosmos/cosmos-sdk/x/auth/vesting"
+	supplyexported "github.com/cosmos/cosmos-sdk/x/supply/exported"
+	"github.com/kava-labs/kava/x/validator-vesting/internal/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+// NewQuerier returns a new querier function
+func NewQuerier(keeper Keeper) sdk.Querier {
+	return func(ctx sdk.Context, path []string, req abci.RequestQuery) (res []byte, err sdk.Error) {
+		switch path[0] {
+		case types.QueryCirculatingSupply:
+			return queryGetCirculatingSupply(ctx, req, keeper)
+		default:
+			return nil, sdk.ErrUnknownRequest("unknown cdp query endpoint")
+		}
+	}
+}
+
+func queryGetCirculatingSupply(ctx sdk.Context, req abci.RequestQuery, keeper Keeper) ([]byte, sdk.Error) {
+	circulatingSupply := sdk.ZeroInt()
+	keeper.ak.IterateAccounts(ctx,
+		func(acc authexported.Account) (stop bool) {
+			// exclude module account
+			_, ok := acc.(supplyexported.ModuleAccountI)
+			if ok {
+				return false
+			}
+
+			// periodic vesting account
+			vacc, ok := acc.(vesting.PeriodicVestingAccount)
+			if ok {
+				balance := vacc.GetCoins().AmountOf("ukava")
```

Comment:
I think the "ukava" coin denom shouldn't be hard coded.
Possible options:
- get the denom from another module, ie `stakingKeeper.BondDenom(ctx)`
- just return all coins, and leave it up to the user to parse out the coins they're interested in